generators/client/templates/angular/package.json.ejs

```diff
@@ -46,7 +46,7 @@
     <%_ } _%>
     "core-js": "3.1.3",
     "moment": "2.24.0",
-    "ng-jhipster": "0.9.3",
+    "ng-jhipster": "0.10.0",
```

Comment:
I was about to raise, but, hold my changes as I want to downgrade `ng-bootstrap` in `ng-jhipster` to fix warnings during install.

Reply:
Oh I see, so let's close this PR. Ping me as soon as you do the PR to ng-jhipster and need a new release. I will be able to do it

Reply:
done